src/sentry/lang/native/sources.py

```diff
@@ -252,6 +256,11 @@ def override_key(schema: dict, keys_to_redact: list[str]) -> None:
 
 LAST_UPLOAD_TTL = 24 * 3600
 
+TOKEN_TTL_SECONDS = 3600
+# Set the TTL a little shorter than the actual token expiration to acomodate a bit of delay
+# while obtaining the token and storing it
+token_cache: TTLCache[Any, Any] = TTLCache(maxsize=128, ttl=TOKEN_TTL_SECONDS - 100)
```

Comment:
I think we should have at least 5 minutes, as this is the timeout of symbolicator, probably be on the safe side and use 10 minutes.